src/Agent.Plugins/Agent.Plugins.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Agent.Sdk\Agent.Sdk.csproj" />
+    <ProjectReference Include="..\Microsoft.VisualStudio.Services.Agent\Microsoft.VisualStudio.Services.Agent.csproj" />
```

Comment:
why you need this?

Reply:
Constants.Variables.System.TeamProjectId

Reply:
remove the dependency, use "System.TeamProjectId" for now, i will fix this later. :)

Reply:
done. Added a TODO for you :)